[17.0][ADD] mail_bounce (#16)
trisdoan/social: mail_prevent_bounce_loop/readme/DESCRIPTION.md

> This module helps to prevent feedback loop between odoo and

Reviewer: ```suggestion
This module helps to prevent infinite loop between odoo and
```

autoresponder, which is currently raised in [here](<https://github.com/odoo/odoo/issues/132666>)

> - For example, when odoo sends an invoice email to customer mail address, autoresponder

Reviewer: ```suggestion
- Case 1: odoo sends an invoice email to customer mail address, autoresponder
```

> triggers then it sends auto-reply mail (without enough information for

Reviewer: ```suggestion
 sends auto-reply mail (without enough information for
```

> odoo to detect) to catchall/unroutable email in odoo. Odoo receives and

Reviewer: ```suggestion
odoo to detect it as an auto-reply: headers `[insert here concerned headers + link to odoo code with permalink]`) to catchall/unroutable email in odoo. Odoo receives and
```

send another one, resulting ping-pong situation.

> - Another example resulting feedback loop is by a cron, Notification: Send scheduled

Reviewer: ```suggestion
- Case 2: a cron, Notification: Send scheduled
```
